loader/src/main/java/com/megaease/easeagent/Main.java

```diff
@@ -63,6 +66,12 @@ public static void premain(final String args, final Instrumentation inst) throws
             urls.addAll(directoryPluginUrls(p));
         }
 
+        // find log4j2.xml in the same directory
+        File logConf = new File(jar.getParent() + File.separator + defaultAgentLogConf);
+        if (logConf.exists()) {
+            defaultAgentLogConf = logConf.getAbsolutePath();
+        }
+
```

Comment:
Do not make any changes to the static defaultAgentLogConf. It is more appropriate to move this part of the logic to the getLogConfigPath() method.